Calculator.php

```diff
@@ -161,10 +152,20 @@ function show_history($history)
         info('You have no history');
     } else {
         $historyGroups = array_group($history, 'date');
+        $historyCommands = ['full'];
+
+        foreach ($historyGroups as $mainDate => $dateGroup) {
+            $historyCommands[] = $mainDate;
+        }
+
+        $showDateHistory = choose('Enter date of history (date - month - year) or "full" : ', $historyCommands);
```

Comment:
it is not user friendly to show each available date as an option to choose (imagine how it will works if we'll have records for each of 165 days by one year only!)

I suggest to union command description from the help block with this message and write something like this: "Please, choose the date to export in DD-MM-YYY format (e.g. 25-12-2022) or type "full" to show all history."